polaris-service/src/main/java/org/apache/polaris/service/catalog/ExtendedConfigResponse.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.polaris.service.catalog;
+
+import com.google.common.base.MoreObjects;
+import java.util.List;
+import java.util.Map;
+import org.apache.iceberg.rest.responses.ConfigResponse;
+
+// TODO: Replace with Iceberg's ConfigResponse after integrating Iceberg 1.7.0
+// Related PR: https://github.com/apache/iceberg/pull/10929#issuecomment-2418591566
```

Comment:
Does it make sense to hold off until 1.7.0?

Reply:
Yes, we will hold if off until 17.0 based on the discussion here: https://github.com/apache/polaris/issues/316